### PR TITLE
python312Packages.faiss: skip broken distutils branch on aarch64-linux

### PR DESCRIPTION
**Pull request by GaetanLepage**
## Things done

https://github.com/facebookresearch/faiss/issues/3936

- **faiss: cleaning**
- **python312Packages.faiss: mark as broken on aarch64-linux**

cc @SomeoneSerge

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/python-modules/faiss/default.nix [hunk 1]
```diff
@@ -50,6 +50,15 @@ buildPythonPackage {
...
     (lib.flip builtins.removeAttrs [ "mainProgram" ])
-    (m: m // { description = "Bindings for faiss, the similarity search library"; })
+    (
+      m:
+      m
+      // {
+        description = "Bindings for faiss, the similarity search library";
+        # Attempts at importing deprecated and removed `numpy.distutils.cpuinfo` to check for SVE support
+        # https://github.com/facebookresearch/faiss/issues/3936
+        broken = stdenv.hostPlatform.isLinux stdenv.hostPlatform.isAarch64 && pythonAtLeast "3.12";
```
Comment (SomeoneSerge) [inline pkgs/development/python-modules/faiss/default.nix:60]: Maybe `m.broken || ...`

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 347735`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 3 packages marked as broken and skipped:</summary>
  <ul>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>sqlite-vss</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 30 packages built:</summary>
  <ul>
    <li>faiss</li>
    <li>faiss.dist</li>
    <li>faissWithCuda</li>
    <li>faissWithCuda.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.colbert-ai</li>
    <li>python311Packages.colbert-ai.dist</li>
    <li>python311Packages.faiss</li>
    <li>python311Packages.faiss.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pytorch-metric-learning</li>
    <li>python311Packages.pytorch-metric-learning.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.autofaiss</li>
    <li>python312Packages.autofaiss.dist</li>
    <li>python312Packages.colbert-ai</li>
    <li>python312Packages.colbert-ai.dist</li>
    <li>python312Packages.faiss</li>
    <li>python312Packages.faiss.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-metric-learning</li>
    <li>python312Packages.pytorch-metric-learning.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 5 packages marked as broken and skipped:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>sqlite-vss</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 28 packages built:</summary>
  <ul>
    <li>faiss</li>
    <li>faiss.dist</li>
    <li>faissWithCuda</li>
    <li>faissWithCuda.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.colbert-ai</li>
    <li>python311Packages.colbert-ai.dist</li>
    <li>python311Packages.faiss</li>
    <li>python311Packages.faiss.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pytorch-metric-learning</li>
    <li>python311Packages.pytorch-metric-learning.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.autofaiss</li>
    <li>python312Packages.autofaiss.dist</li>
    <li>python312Packages.colbert-ai</li>
    <li>python312Packages.colbert-ai.dist</li>
    <li>python312Packages.faiss</li>
    <li>python312Packages.faiss.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-metric-learning</li>
    <li>python312Packages.pytorch-metric-learning.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:fast_forward: 5 packages marked as broken and skipped:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>sqlite-vss</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 20 packages built:</summary>
  <ul>
    <li>faiss</li>
    <li>faiss.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.colbert-ai</li>
    <li>python311Packages.colbert-ai.dist</li>
    <li>python311Packages.faiss</li>
    <li>python311Packages.faiss.dist</li>
    <li>python311Packages.pytorch-metric-learning</li>
    <li>python311Packages.pytorch-metric-learning.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.autofaiss</li>
    <li>python312Packages.autofaiss.dist</li>
    <li>python312Packages.colbert-ai</li>
    <li>python312Packages.colbert-ai.dist</li>
    <li>python312Packages.faiss</li>
    <li>python312Packages.faiss.dist</li>
    <li>python312Packages.pytorch-metric-learning</li>
    <li>python312Packages.pytorch-metric-learning.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 3 packages marked as broken and skipped:</summary>
  <ul>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>sqlite-vss</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 28 packages built:</summary>
  <ul>
    <li>faiss</li>
    <li>faiss.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.colbert-ai</li>
    <li>python311Packages.colbert-ai.dist</li>
    <li>python311Packages.faiss</li>
    <li>python311Packages.faiss.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pytorch-metric-learning</li>
    <li>python311Packages.pytorch-metric-learning.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.autofaiss</li>
    <li>python312Packages.autofaiss.dist</li>
    <li>python312Packages.colbert-ai</li>
    <li>python312Packages.colbert-ai.dist</li>
    <li>python312Packages.faiss</li>
    <li>python312Packages.faiss.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-metric-learning</li>
    <li>python312Packages.pytorch-metric-learning.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

File: pkgs/development/libraries/science/math/faiss/default.nix [hunk 2]
```diff
@@ -48,11 +48,16 @@ stdenv.mkDerivation {
...
-    repo = pname;
-    rev = "v${version}";
+    repo = "faiss";
+    rev = "refs/tags/v${version}";
     hash = "sha256-P8TynU6jz5NbcWLdI7n4LX5Gdz0Ks72bmOzQ3LGjQCQ=";
   };
 
+  postPatch = lib.optionalString pythonSupport ''
+    substituteInPlace faiss/python/loader.py \
+      --replace-fail "import numpy.distutils.cpuinfo" "return False"
```
Comment (SomeoneSerge) [inline pkgs/development/libraries/science/math/faiss/default.nix:58]: THere are two matches in `loader.py`, and the 2nd one in a function that, I think, is meant to return a dict so this doesn't type check. Maybe we replace `# platform-dependent legacy fallback using numpy.distutils.cpuinfo` with the return statement? xD

Comment (GaetanLepage) [reply pkgs/development/libraries/science/math/faiss/default.nix]: Sure, I had observed this, but didn't think that it was too much of an issue.
Your idea is indeed better.
I pushed it.

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 347735`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 3 packages marked as broken and skipped:</summary>
  <ul>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>sqlite-vss</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 30 packages built:</summary>
  <ul>
    <li>faiss</li>
    <li>faiss.dist</li>
    <li>faissWithCuda</li>
    <li>faissWithCuda.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.colbert-ai</li>
    <li>python311Packages.colbert-ai.dist</li>
    <li>python311Packages.faiss</li>
    <li>python311Packages.faiss.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pytorch-metric-learning</li>
    <li>python311Packages.pytorch-metric-learning.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.autofaiss</li>
    <li>python312Packages.autofaiss.dist</li>
    <li>python312Packages.colbert-ai</li>
    <li>python312Packages.colbert-ai.dist</li>
    <li>python312Packages.faiss</li>
    <li>python312Packages.faiss.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-metric-learning</li>
    <li>python312Packages.pytorch-metric-learning.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 5 packages marked as broken and skipped:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>sqlite-vss</li>
  </ul>
</details>
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 26 packages built:</summary>
  <ul>
    <li>faiss</li>
    <li>faiss.dist</li>
    <li>faissWithCuda</li>
    <li>faissWithCuda.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.colbert-ai</li>
    <li>python311Packages.colbert-ai.dist</li>
    <li>python311Packages.faiss</li>
    <li>python311Packages.faiss.dist</li>
    <li>python311Packages.pytorch-metric-learning</li>
    <li>python311Packages.pytorch-metric-learning.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.autofaiss</li>
    <li>python312Packages.autofaiss.dist</li>
    <li>python312Packages.colbert-ai</li>
    <li>python312Packages.colbert-ai.dist</li>
    <li>python312Packages.faiss</li>
    <li>python312Packages.faiss.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-metric-learning</li>
    <li>python312Packages.pytorch-metric-learning.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:fast_forward: 5 packages marked as broken and skipped:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>sqlite-vss</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 20 packages built:</summary>
  <ul>
    <li>faiss</li>
    <li>faiss.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.colbert-ai</li>
    <li>python311Packages.colbert-ai.dist</li>
    <li>python311Packages.faiss</li>
    <li>python311Packages.faiss.dist</li>
    <li>python311Packages.pytorch-metric-learning</li>
    <li>python311Packages.pytorch-metric-learning.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.autofaiss</li>
    <li>python312Packages.autofaiss.dist</li>
    <li>python312Packages.colbert-ai</li>
    <li>python312Packages.colbert-ai.dist</li>
    <li>python312Packages.faiss</li>
    <li>python312Packages.faiss.dist</li>
    <li>python312Packages.pytorch-metric-learning</li>
    <li>python312Packages.pytorch-metric-learning.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 3 packages marked as broken and skipped:</summary>
  <ul>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>sqlite-vss</li>
  </ul>
</details>
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>faiss</li>
    <li>faiss.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.colbert-ai</li>
    <li>python311Packages.colbert-ai.dist</li>
    <li>python311Packages.faiss</li>
    <li>python311Packages.faiss.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pytorch-metric-learning</li>
    <li>python311Packages.pytorch-metric-learning.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.autofaiss</li>
    <li>python312Packages.autofaiss.dist</li>
    <li>python312Packages.colbert-ai</li>
    <li>python312Packages.colbert-ai.dist</li>
    <li>python312Packages.faiss</li>
    <li>python312Packages.faiss.dist</li>
    <li>python312Packages.pytorch-metric-learning</li>
    <li>python312Packages.pytorch-metric-learning.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 347735`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 3 packages marked as broken and skipped:</summary>
  <ul>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>sqlite-vss</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 30 packages built:</summary>
  <ul>
    <li>faiss</li>
    <li>faiss.dist</li>
    <li>faissWithCuda</li>
    <li>faissWithCuda.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.colbert-ai</li>
    <li>python311Packages.colbert-ai.dist</li>
    <li>python311Packages.faiss</li>
    <li>python311Packages.faiss.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pytorch-metric-learning</li>
    <li>python311Packages.pytorch-metric-learning.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.autofaiss</li>
    <li>python312Packages.autofaiss.dist</li>
    <li>python312Packages.colbert-ai</li>
    <li>python312Packages.colbert-ai.dist</li>
    <li>python312Packages.faiss</li>
    <li>python312Packages.faiss.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-metric-learning</li>
    <li>python312Packages.pytorch-metric-learning.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:fast_forward: 5 packages marked as broken and skipped:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>sqlite-vss</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 28 packages built:</summary>
  <ul>
    <li>faiss</li>
    <li>faiss.dist</li>
    <li>faissWithCuda</li>
    <li>faissWithCuda.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.colbert-ai</li>
    <li>python311Packages.colbert-ai.dist</li>
    <li>python311Packages.faiss</li>
    <li>python311Packages.faiss.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pytorch-metric-learning</li>
    <li>python311Packages.pytorch-metric-learning.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.autofaiss</li>
    <li>python312Packages.autofaiss.dist</li>
    <li>python312Packages.colbert-ai</li>
    <li>python312Packages.colbert-ai.dist</li>
    <li>python312Packages.faiss</li>
    <li>python312Packages.faiss.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-metric-learning</li>
    <li>python312Packages.pytorch-metric-learning.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:fast_forward: 5 packages marked as broken and skipped:</summary>
  <ul>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>sqlite-vss</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 20 packages built:</summary>
  <ul>
    <li>faiss</li>
    <li>faiss.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.colbert-ai</li>
    <li>python311Packages.colbert-ai.dist</li>
    <li>python311Packages.faiss</li>
    <li>python311Packages.faiss.dist</li>
    <li>python311Packages.pytorch-metric-learning</li>
    <li>python311Packages.pytorch-metric-learning.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.autofaiss</li>
    <li>python312Packages.autofaiss.dist</li>
    <li>python312Packages.colbert-ai</li>
    <li>python312Packages.colbert-ai.dist</li>
    <li>python312Packages.faiss</li>
    <li>python312Packages.faiss.dist</li>
    <li>python312Packages.pytorch-metric-learning</li>
    <li>python312Packages.pytorch-metric-learning.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:fast_forward: 3 packages marked as broken and skipped:</summary>
  <ul>
    <li>python312Packages.txtai</li>
    <li>python312Packages.txtai.dist</li>
    <li>sqlite-vss</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 28 packages built:</summary>
  <ul>
    <li>faiss</li>
    <li>faiss.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>python311Packages.autofaiss</li>
    <li>python311Packages.autofaiss.dist</li>
    <li>python311Packages.colbert-ai</li>
    <li>python311Packages.colbert-ai.dist</li>
    <li>python311Packages.faiss</li>
    <li>python311Packages.faiss.dist</li>
    <li>python311Packages.pyannote-audio</li>
    <li>python311Packages.pyannote-audio.dist</li>
    <li>python311Packages.pytorch-metric-learning</li>
    <li>python311Packages.pytorch-metric-learning.dist</li>
    <li>python311Packages.txtai</li>
    <li>python311Packages.txtai.dist</li>
    <li>python312Packages.autofaiss</li>
    <li>python312Packages.autofaiss.dist</li>
    <li>python312Packages.colbert-ai</li>
    <li>python312Packages.colbert-ai.dist</li>
    <li>python312Packages.faiss</li>
    <li>python312Packages.faiss.dist</li>
    <li>python312Packages.pyannote-audio</li>
    <li>python312Packages.pyannote-audio.dist</li>
    <li>python312Packages.pytorch-metric-learning</li>
    <li>python312Packages.pytorch-metric-learning.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

**Pull request merged**